### PR TITLE
Stim info view

### PR DESCRIPTION
**Pull request by ivanmauricio**
* Style StimInfo page

### CODE REVIEW

File: src/elm/Views/StimInfo.elm [hunk 1]
```diff
@@ -1,10 +1,46 @@
...
+                , p [ classes [ bodyFont ] ] [ text <| model.selectedStim.instructions ]
+                ]
+            ]
+        , div
+            [ backgroundImageCover "./assets/AddStim/zigzag_stim_how_to_bg.svg"
+            , class "flex flex-column items-center"
+            ]
+            [ img [ src "./assets/StimInfo/add_video_icn.svg", class "pt4" ] []
+            , h1 [ classes [ headerFont, "mb3" ] ] [ text "Can this video help?" ]
+            , div [ class "video-height" ]
```
Comment (maxgerber) [inline src/elm/Views/StimInfo.elm]: Can we put a viewIf clause here to remove if it doesn't exist?

File: src/elm/Views/StimInfo.elm [hunk 2]
```diff
@@ -1,10 +1,46 @@
...
+                [ div [ class "absolute left-0 ml3 pointer", onClick (NavigateTo StimPreparation) ] [ img [ src "./assets/StimInfo/close_btn_white.svg" ] [] ]
+                , h1 [ classes [ headerFont, "white pt3 mb3" ] ] [ text <| model.selectedStim.stimName ]
+                ]
+            , button
+                [ classes
+                    [ "green br-pill pa1 pointer w5 mb4 bg-white bn"
+                    , bodyFont
+                    ]
+                ]
+                [ text <| bodyPartToString model.selectedStim.bodyPart ]
```
Comment (maxgerber) [inline src/elm/Views/StimInfo.elm]: Can we replace this function with union type to string?

File: src/css/_custom.css [hunk 3]
```diff
@@ -335,3 +335,7 @@ body {
 .fill-screen {
   min-height: 100vh;
 }
+
+.video-height {
```
Comment (maxgerber) [inline src/css/_custom.css]: What's the thinking behind making this a pixel-height? Sure that there is one but just curious ;)

**Comment by ivanmauricio** [comment]
@maxgerber Made all the changes!

**Pull request merged**